libs/chain-signatures/contract/src/storage_keys.rs

```diff
@@ -10,7 +10,7 @@ pub enum StorageKey {
     /// Proposed updates to the contract code and config.
     _DeprecatedProposedUpdatesEntries,
     _DeprecatedRequestsByTimestamp,
-    /// Pending signature requests.
+    /// Pending requests.
```

Comment:
need to add a storage key for the ckd requests (at the end of the enum).
https://docs.near.org/smart-contracts/anatomy/collections#instantiation